### ISSUE TITLE
Images do not show up with title

### ISSUE THREAD

**Issue by mehedihasan619** [extent-framework/extentreports-java#260]
Issue Description: Images added using `getScreenShotPath(testCaseName,driver),"Title")` are missing the title when they show up in Extent Report.

Reproduction Code Sample:

```
		try {
			driver =(WebDriver)result.getTestClass().getRealClass().getDeclaredField("driver").get(result.getInstance());
			extentTestThread.get().addScreenCaptureFromPath(getScreenShotPath(testCaseName,driver),"Title");
		} catch (Exception e) {
			//Exception handled
		}
```

Screenshot: https://prnt.sc/uvqfow

Extent Report Version:  4.1.4
Language: Java

**Comment by mehedihasan619** [comment]
Hi @anshooarora,
Would you mind looking at this?

**Comment by mehedihasan619** [comment]
The problem is resolved after upgrading to version 5.0.4.
